about.md

```diff
@@ -4,10 +4,6 @@ title: About
 permalink: /about/
 ---
 
-<center><p>
```

Comment:
We must have accidentally removed this, re-adding this section back instead of removing it.